### PR TITLE
Backend Test for Version Endpoint

### PR DESCRIPTION
**Pull request by galeza**
Draft PR for code review
Branch created by Selby. I trigger only PR.

- [] I have checked that no one else is working on similar changes.
- [ ] I have read the latest [**README**](README.md) and understand the development workflow.
- [ ] The name of this branch is: **`feature/<branch_name>`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [ ] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [ ] There are no merge conflicts.
- [ ] There are no console warnings and errors.
- [ ] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [ ] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

**Comment by galeza** [comment]
@Selbinyyaz 

I created a draft PR to make review easier

File: mocha/test/version.test.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+const apiEndPoints = require("../utils/apiEndpoints");
+const apiHelper = require("../utils/apiHelper");
+const validator = require("../utils/validators");
+const testData = require("../utils/testData");
+
+let apiEndPoint= apiEndPoints.versionApiEndpoint;
+
+describe("Get Version endpoints test", function(){
+    it("user enters to see version",async function(){
+            let response= await apiHelper.sendPOSTRequest(
```
Comment (galeza) [inline mocha/test/version.test.js]: @Selbinyyaz 
This is a GET endpoint so we don't use POST but sendGETRequest.
I think that if you rebase or merge from staging you will see this method.
this is the reason why your test doesn't work

File: mocha/test/version.test.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+            APP_URL,
+            apiEndPoint
+    );
+    validator.validateStatusCodeAndOk(
+        response,
```
Comment (galeza) [inline mocha/test/version.test.js:17]: Formatting - do you have Prettier installed for your IDE like VSC?
If not look here: https://github.com/FightPandemics/FightPandemics/pull/1717#issuecomment-719582599

File: mocha/utils/validators.js [hunk 3]
```diff
@@ -23,3 +23,16 @@ exports.validateStatusCodeAndSuccess = function (
     .to.have.a.property("success")
     .to.be.equal(successMessage);
 };
+
+exports.validateStatusCodeAndOk = function (
```
Comment (galeza) [inline mocha/utils/validators.js:44]: I think that we will have to add more generic validator that takes dict: key-value and validate it.

**Review by galeza — COMMENTED** [review]
@Selbinyyaz I added some comments and explanation why test fails

File: mocha/test/version.test.js (hunk 2 above)
Comment (Selbinyyaz) [reply mocha/test/version.test.js:17]: Thank you, Agnieszka. I don't have Prettier but I will install it.

**Comment by adorapham** [comment]
@Selbinyyaz @galeza , since this pull request has a lot of difference from the staging. I will close this one and pull the latest staging code

**Pull request closed without merging**